src/components/views/dialogs/UserSettingsDialog.tsx

```diff
@@ -37,6 +37,20 @@ import SidebarUserSettingsTab from "../settings/tabs/user/SidebarUserSettingsTab
 import KeyboardUserSettingsTab from "../settings/tabs/user/KeyboardUserSettingsTab";
 import { UserTab } from "./UserTab";
 
+/**
+ * Returns null if labs are disabled and no betas exist
+ */
+const getLabsTabLabel = (): string | null => {
+    const showLabs = SdkConfig.get("show_labs_settings");
+    const showBetas = SettingsStore.getFeatureSettingNames().some(k => SettingsStore.getBetaInfo(k));
+
+    if (showBetas && showLabs) return _td("Betas & labs");
```

Comment:
```suggestion
    if (showBetas && showLabs) return _td("Betas & Labs");
```
I would capitalize Labs for consistency with Security & Privacy and Help & About.

Thanks for making this PR ❤️